composer.json

```diff
@@ -12,10 +12,10 @@
         "react/socket-client": "^0.5.0",
         "mkraemer/react-pcntl": "2.0.*",
         "monolog/monolog": "^1.3",
-        "paragonie/random_compat": "^2"
+        "paragonie/random_compat": "^2",
+        "guzzlehttp/psr7": "1.2.3"
```

Comment:
Imho that makes sense to be sure we always get the same dependencies- if using a "locked" vcs alias like I'm doing or if using `react/http`- you have basically already done that for `guzzlehttp` but not yet for `react/socket`?